### DISCUSSION TITLE
How to change the color of the Text?

### DISCUSSION THREAD

**Discussion by henjoe** [frangoteam/FUXA#1274] [Q&A]
Hi I don't see any property to change the color of the text rather than color black? 
Am I missing something? 

![image](https://github.com/user-attachments/assets/c8c53605-28fc-4bb4-9625-36bd1f209b23)

I actually tried this, but seems NOT working. 
![image](https://github.com/user-attachments/assets/aeb8420a-0945-40e6-b29b-f3f32e6b25e6)

Any help is highly appreciated. 

Regards,

**Comment by henjoe** [comment]
This is weird because on the action, it actually works the Blink:

![image](https://github.com/user-attachments/assets/b2112f65-7b02-4aa8-b899-bd630599b5fc)

**Comment by unocelli** [comment]
Hi, should be fixed in recent commits.
